### DISCUSSION TITLE
Error for deploying a WebGL Unity with a file > 100MB

### DISCUSSION THREAD

**Discussion by dacoten** [vercel/vercel#6789] [Help]
Hi everyone,

I have a "*.data" file In WebGL that exceeds the size is greater than 100MB.

I got a problem when I cannot push my source code to the Vercel server.

The error is "File size limit exceeded (100 MB)".

How can I resolve this problem?

Thanks.

**Comment by JurajFIC** [comment]
Hi, go here : https://forum.unity.com/?_ga=2.256561420.605563258.1633680719-2110022891.1633680719      and ask :)
